src/parser.py

```diff
@@ -92,7 +93,8 @@ def get_medias(auth, user_id, include_retweets, image_size, since, since_id, unt
     if 'access_token' in auth and 'access_token_secret' in auth:
         tweepy_auth.set_access_token(auth['access_token'], auth['access_token_secret'])
 
-    api = tweepy.API(tweepy_auth, wait_on_rate_limit=True, wait_on_rate_limit_notify=True)
```

Comment:
wait_on_rate_limit_notify=True
This option is deprecated in current Tweepy releases, and produces an error message, so it is removed.

Reply:
> Remove API.wait_on_rate_limit_notify ([f325738](https://github.com/tweepy/tweepy/commit/f3257389ed566029e7eec0598597afeef6900d35))
> Always log warning when rate limit reached

Indeed 👍